src/test/contractTest/resources/application.properties

```diff
@@ -19,12 +19,12 @@ feign.httpclient.enabled = true
 tornado.url=${TORNADO_URL:http://localhost:8090/rs/render}
 tornado.accessKey=${TORNADO_ACCESS_KEY:}
 
-ccd.data.store.api.url=${CCD_DATA_STORE_API_URL:http://ccd-data-store-api:4452}
+ccd.data-store-api-url=${CCD_DATA_STORE_API_URL:http://ccd-data-store-api:4452}
```

Comment:
maybe all properties files should be refactored into yaml files just for consistency?